docs/operations/Deployment.md

```diff
@@ -1,6 +1,6 @@
 # Deployment
 
-Besides using Kelon as Gateway with a proxy like Envoy or as plugin in a Service-Mesh, it can also be deployed standalone and integrated in your existing services with just a few lines of extra code.
+Besides using Kelon as Gateway with a proxy or as plugin in a Service-Mesh, it can also be deployed standalone and integrated in your existing services with just a few lines of extra code.
```

Comment:
I think first part "Besides using Kelon as Gateway with a proxy or as plugin in a Service-Mesh" should be removed, too?

Reply:
You are totally right. I would go ahead with something like this:
```suggestion
Kelon can be easily integrated in your existing services with just a few lines of extra code by either integrating it in each service or i.e. your API-Gateway.
```